app/javascript/components/mentoring/inbox/conversation-list/Conversation.jsx

```diff
@@ -16,30 +17,54 @@ export function Conversation({
   updatedAt,
   url,
 }) {
+  const [referenceElement, setReferenceElement] = useState(null)
+  const [popperElement, setPopperElement] = useState(null)
+  const [showPopper, setShowPopper] = useState(null)
+  const { styles, attributes } = usePopper(referenceElement, popperElement)
+
   return (
-    <tr>
-      <td>
-        <img
-          style={{ width: 100 }}
-          src={trackIconUrl}
-          alt={`icon for ${trackTitle} track`}
-        />
-      </td>
-      <td>
-        <img
-          style={{ width: 100 }}
-          src={menteeAvatarUrl}
-          alt={`avatar for ${menteeHandle}`}
-        />
-      </td>
-      <td>{menteeHandle}</td>
-      <td>{exerciseTitle}</td>
-      <td>{isStarred.toString()}</td>
-      <td>{haveMentoredPreviously.toString()}</td>
-      <td>{isNewIteration.toString()}</td>
-      <td>{postsCount}</td>
-      <td>{dayjs(updatedAt).fromNow()}</td>
-      <td>{url}</td>
-    </tr>
+    <>
+      <tr
+        ref={setReferenceElement}
+        onMouseEnter={() => {
+          setShowPopper(true)
+        }}
+        onMouseLeave={() => {
+          setShowPopper(null)
+        }}
+      >
+        <td>
+          <img
+            style={{ width: 100 }}
+            src={trackIconUrl}
+            alt={`icon for ${trackTitle} track`}
+          />
+        </td>
+        <td>
+          <img
+            style={{ width: 100 }}
+            src={menteeAvatarUrl}
+            alt={`avatar for ${menteeHandle}`}
+          />
+        </td>
+        <td>{menteeHandle}</td>
+        <td>{exerciseTitle}</td>
+        <td>{isStarred.toString()}</td>
+        <td>{haveMentoredPreviously.toString()}</td>
+        <td>{isNewIteration.toString()}</td>
+        <td>{postsCount}</td>
+        <td>{dayjs(updatedAt).fromNow()}</td>
+        <td>{url}</td>
+      </tr>
+      {showPopper && (
+        <div
+          ref={setPopperElement}
+          style={styles.popper}
+          {...attributes.popper}
+        >
+          Lorem ipsum
+        </div>
```

Comment:
Once we'll have multiple of these popups, we should probably extract it to a separate component. It is perfectly fine for now.